src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -78,11 +93,18 @@ public void periodic() {
                     }
                 }
 
-                if (timer.hasElapsed(2)) {
+                if (noteAtShooterSensor && ! isEjecting) {
+                    timer.stop();
+                    intakeMotor.set(0);
+                    currentState = IntakeState.INTAKING;
+                }
```

Comment:
I'm not 100% sure this code is actually needed. It doesn't quite make sense to me.